packages/cli/src/rollup-plugins/flow-and-prod-dev-entry.ts

```diff
@@ -49,6 +49,7 @@ export default function flowAndNodeDevProdEntry(
       }
 
       if (
+        source.startsWith("\0") ||
```

Comment:
wait - is `\0babelWhatever` added to the top of a generated virtual source code? 🤔 how does it work? kinda blurry for me in a rush

Reply:
`source` is referring to the unresolved module specifier, not source code.

Reply:
Oh, gotcha.